### ISSUE TITLE
GetAtt in Outputs not reporting template invalid

### ISSUE THREAD

**Issue by martysweet** [martysweet/cfn-lint#51]
Template

```yaml
Resources:
  Bucket:
    Type: AWS::S3::Bucket

Outputs:
  DBDNS:
    Description: DNS Name of the DB Instance
    Value: !GetAtt Database.Endpoint.Address
```

Results in a Valid Template, however `Database` is not defined in resources, so this template should be invalid.

**Comment by martysweet** [comment]
This issue is still present in v1.3.4

**Comment by RazzM13** [comment]
Hi @martysweet, this issue should be fixed in the above PR.
